README.adoc

```diff
@@ -76,13 +76,22 @@ tBTC's first 6-week audit was completed by ConsenSys Diligence on March 27,
 They've published a detailed https://diligence.consensys.net/audits/2020/02/thesis-tbtc-and-keep/[audit report]
 and https://diligence.consensys.net/audits/2020/03/thesis-cryptographic-review/[cryptographic review].
 
-tBTC is currently under audit by two additional parties—
-https://www.trailofbits.com/[Trail of Bits] and an unannounced Bitcoin-focused
-security researcher.
-
-Review a more focused treatment of tBTC's security model https://tbtc.network/developers/tbtc-security-model/[here].
-
-Please report any security issues you find to mailto:security@keep.network[security@keep.network].
+An additional audit was conducted by Bitcoin-focused security researcher
+https://twitter.com/sr_gi[Sergi Delgado] from May 25 to May 31, 2020. You can
+https://srgi.me/resources/reports/tbtc_audit.pdf[review Sergi's results on his site.].
```

Comment:
There's an extra dot at the end of this line.

Reply:
If anyone comments that my regex syntax in that last commit is off because `]` is used for character sets and `.` is for any character, I just don't know what I'll do :joy: